src/index.html

```diff
@@ -11,12 +11,104 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_list">
+      <a
+        href="https://mate.academy.com.ua"
+        class="moyo_logo"
+      >
+        <img
+          alt="Moyo logo"
+          src="images/logo.png"
+        />
+      </a>
+      <nav class="nav_list">
+        <ul class="ul_list">
+          <li>
+            <a
+              href="#1"
+              class="a_style is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="#2"
+              class="a_style"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="#3"
+              class="a_style"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              href="#4"
+              class="a_style"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
```

Comment:
```suggestion
          <li>
            <a
              href="#4"
              class="a_style"
              data-qa="hover"
            >
              LAPTOPS & COMPUTERS
            </a>
          </li>
          
          <li>
```

add an empty line between multiple sibling blocks of html